samcli/local/services/base_local_service.py

```diff
@@ -118,4 +121,19 @@ def get_lambda_output(stdout_stream):
             # Last line is Lambda response. Make sure to strip() so we get rid of extra whitespaces & newlines around
             lambda_response = stdout_data[last_line_position:].strip()
 
-        return lambda_response, lambda_logs
+        is_lambda_user_error_response = False
+
+        try:
```

Comment:
Can you move this to a separate method for readability & testability? This whole blob is only to check if this is an error response.

Reply:
Can do :)